src/chunk_constraint.c

```diff
@@ -889,6 +897,47 @@ chunk_constraint_rename_hypertable_tuple(TupleInfo *ti, void *data)
 	return true;
 }
 
+static bool
+chunk_constraint_alter_hypertable_tuple(TupleInfo *ti, void *data)
+{
+	AlterHypertableConstraintInfo *info = data;
+
+	bool		nulls[Natts_chunk_constraint];
+	Datum		values[Natts_chunk_constraint];
+	Name		chunk_constraint_name;
+	Constraint *cmd_con;
+
+
+	heap_deform_tuple(ti->tuple, ti->desc, values, nulls);
+	chunk_constraint_name = DatumGetName(values[Anum_chunk_constraint_constraint_name - 1]);
+
+	cmd_con = castNode(Constraint, info->cmd->def);
+
+	cmd_con->conname = NameStr(*chunk_constraint_name);
+	AlterTableInternal(info->chunk_relid, list_make1(info->cmd), false);
+	return true;
+}
+
+static bool
+chunk_constraint_validate_hypertable_tuple(TupleInfo *ti, void *data)
```

Comment:
These functions are almost identical, save for one or two lines. It must be possible to reduce a bunch of code duplication here, e.g., by having one function and a small if-else clause within based on subtype of the command.